src/workerd/api/actor-state.c++

```diff
@@ -660,6 +660,22 @@ jsg::Promise<jsg::Value> DurableObjectStorage::transaction(jsg::Lock& js,
   });
 }
 
+jsg::Promise<void> DurableObjectStorage::flush(jsg::Lock& lock) {
```

Comment:
Nit: by convention it should be `jsg::Lock& js`

Reply:
I'm guessing that the `Lock` is in effect the direct interface to js land?

Reply:
Effectively yes. It is essentially a wrapper around the v8::Isolate and represents the current lock we have on it.